README.md

```diff
@@ -1,59 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello_Micoverse
 
-> One paragraph statement about the project.
+> First project to get familliar with best practices and tools in software development.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./Hello_world.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML/CSS
+- LINTER
+- VSCODE
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- Github
+- IDE
 
+### Setup
 
+- git clone git@github.com:thenarain/Hello-World.git
```

Comment:
- [ ] (OPTIONAL) This is optional but I strongly suggest that you that you write the correct command for cloning your project which is `git clone https://github.com/thenarain/Hello-World.git`

Reply:
Ok, I will follow the suggestion.